python/snewpy/snowglobes_interface.py

```diff
@@ -156,6 +159,8 @@ def _compute_rates(self, detector, material, flux_file:Path):
         TargetMass = self.detectors[detector].tgt_mass
         data = {}
         energies = np.linspace(7.49e-4, 9.975e-2, 200) # Use the same energy grid as SNOwGLoBES
+        if '_he' in detector:
+            energies = np.linspace(7.49e-4, 2*9.975e-2, 400) #SNOwGLoBES grid for he configurations
```

Comment:
Should the upper edge really be 2*99.75 = 199.5 MeV here? Or should it be 199.75 MeV? (I’d always assumed that the reason for the 0.75 was simply that this is the midpoint of a 0.5 MeV bin; but I can’t see this actually documented anywhere.)

Reply:
Yes, you're right, it should indeed be the bin center, with 0.5MeV bins (snowglobes choice), I will modify that (since the bins are twice, from 200 to 400, I assumed a x2 for the upper edge was correct). Indeed, there is no documentation on this. SimpleRate just kept here the choice of the binning by snowglobes, and when moving from root histograms to python arrays, kept the bin center to define the points in the list.